core/src/Cuda/Kokkos_Cuda_KernelLaunch.hpp

```diff
@@ -34,6 +42,7 @@ __device__ __constant__ unsigned long kokkos_impl_cuda_constant_memory_buffer
     [Kokkos::Impl::CudaTraits::ConstantMemoryUsage / sizeof(unsigned long)];
 
 #endif
+#endif  // KOKKOS_IMPL_CUDA_USE_GRID_CONSTANT
```

Comment:
```suggestion
#endif  // !KOKKOS_IMPL_CUDA_USE_GRID_CONSTANT
```
Maybe this is too trivial, but if I'm reading up from here I would think this was defined in the case of `KOKKOS_IMPL_CUDA_USE_GRID_CONSTANT`